tests/sentry/grouping/test_builtin_fingerprinting.py

```diff
@@ -650,3 +650,32 @@ def test_built_in_hydration_rules_disabled(self):
         event = self.store_event(data=self.hydration_error_trace, project_id=self.project)
         assert event.data.data["fingerprint"] == ["my-route", "{{ default }}"]
         assert event.data.data.get("_fingerprint_info") is None
+
+    @with_feature("organizations:grouping-built-in-fingerprint-rules")
+    def test_built_in_hydration_rules_no_transactions(self):
+        """
+        With the flag enabled, hydration errors with no transactions should work as expected.
+        """
+
+        data_transaction_no_tx = self.hydration_error_trace.copy()
```

Comment:
Should this be a fixture?

Reply:
it could, but it's only used in the django TestCase, not in the pytest tests